pypesto/visualize/observable_mapping.py

```diff
@@ -184,15 +536,19 @@ def plot_splines_from_inner_result(
         The simulated model output.
     observable_ids:
         The ids of the observables.
+    axes:
+        The axes to plot the estimated spline approximations on.
+    rel_and_semiquant_obs_indices:
+        The indices of the relative and semi-quantitative observables in the
+        amici model. Important if both relative and semi-quantitative observables
+        will be plotted. If None, only the semi-quantitative observables will be plotted.
```

Comment:
Only semi-quantitative will be plotted in this method regardless, right?

Reply:
True, it's ambigious.
Changed to:
`The indices of the relative and semi-quantitative observables in the
        amici model. Important if both relative and semi-quantitative observables
        will be plotted on the same axes.` in both places